src/ics/converter/types/calendar.py

```diff
@@ -62,4 +63,24 @@ def serialize(self, component: Component, output: Container, context: ContextDic
         context["VTIMEZONES_AFTER"] = len(output)
 
 
+class CalendarNameConverter(GenericConverter):
```

Comment:
I suppose we can then remove all changes to this file then?

Sorry I did not get the `optional` validator. I had trouble to understand how these were working together.

Reply:
> I suppose we can then remove all changes to this file then?

yes, please do so

Reply:
But then we don't have `X-WR-CALNAME` at all if we do this. Shouldn't we try to always add `X-WR-CALNAME`?

Reply:
I would have this in `CalendarNameConverter.populate` 

```
if item.name == "NAME":
    component.extra_params["X-WR-CALNAME"] = item.value
    return True
```

Reply:
I can make the tests pass, but I am probably not doing it right, you're probably better of doing this yourself at this point 😄